build/ghost-buster/Dockerfile

```diff
@@ -1,33 +1,35 @@
-FROM alpine:3.4
+FROM alpine:3.6
 
 RUN \
 #major part
-  apk --update --no-progress add nodejs-lts \
+  apk --update --no-progress add nodejs-lts nodejs-npm \
 #system utility
-  unrar bash unzip inotify-tools curl wget\
+  unrar bash unzip inotify-tools curl wget sudo \
 #python and other stuff
   py-setuptools python-dev libxml2-dev libxslt-dev py-pip py-libxslt py-lxml
 
 # Install Ghost
-ENV GHOST_VERSION 0.8.0
+ENV GHOST_CLI_VERSION 1.1.1
+ENV GHOST_VERSION 1.8.6
+ENV DEFAULT_BLOG_DOMAIN http://my-ghost-blog.com
 
 RUN \
-  cd /tmp && \
-  curl -sSL "https://ghost.org/archives/ghost-${GHOST_VERSION}.zip" -o ghost.zip && \
-  unzip ghost.zip -d /ghost && \
-  rm -f ghost.zip && \
-  echo "unzip completed" ; \
+  adduser ghost -D -h /ghost -u 987
+RUN \
   cd /ghost && \
-#Fixing version of sqlite3
-  sed -i.bak 's/  "sqlite3".*/    "sqlite3": "3.1.4",/' package.json ;\
-#finaly installing all that stuff
-  npm install --production -d && \
-  npm cache clean
-
+  npm install -g ghost-cli@${GHOST_CLI_VERSION} && \
+  ghost install \
+    --no-start --no-stack \
```

Comment:
Don’t start ghost after installation. Don’t check stack (OS and pre-requisites like mysql or systemd).